.github/workflows/license-check.yml

```diff
@@ -0,0 +1,51 @@
+name: 3PP License Check
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron: '0 4 * * *' # Runs every day at 4am: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule
```

Comment:
Do we really a cron for this?

Reply:
Do we need it for the `build` workflow? 
I think the main reason is so that, on a day where no commit were merged, we'll still get an idea that the various CI steps pass or not. e.g. if a change in dependency happened under our feet and breaks something (or adds new dependencies/versions that are not license-approved)

Reply:
Nothing changes dependencies under our feet. The only time this happens is when we do `yarn upgrade`.

But maybe it would help to notice updates for untracked dependencies suddenly added to the license registry?